cmake/make_versioncpp.py

```diff
@@ -21,7 +21,7 @@
     "boost_locale",
     "boost_log",
     "boost_log_setup",
-    "boost_python",
+    "boost_python%d%d" % (sys.version_info.major, sys.version_info.minor),
```

Comment:
See [my reply](https://github.com/o01eg/freeorion/pull/1#issuecomment-593091618) in the comment section of my PR against the base branch.

If you make sure the version of the bundled Python executable and the boost Python library required by the FO application match, then this fix is better than mine. If not, then this will not work.

Reply:
For Windows cmake ensures it by using `${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}` to choose `boost::python` component.
For MSVS project it's up to user to sync `python*.exe` version in `Common.vcxproj` and `include/pyhon*/` in `FreeOrionD.vcxproj` and `FreeOrionCA.vcxproj`

Reply:
The MSVS project is maintained in the repo, the SDK is provided by us, so it is our responsibility that the correct Python executable is called in the custom build steps for `util\Version.cpp.in`, meaning the MSVS project should be adjusted accordingly.

I propose to drop Python2 entirely anyway, is there any reason left at all to keep it?

Reply:
@Vezzra I suppose we need python2 to preserve it in case to release 0.4.9.1.

If not it doesn't hard to remove python2 from SDK.

Reply:
@Vezzra I've opened PR: https://github.com/freeorion/freeorion-sdk/pull/65

Reply:
Considering that we try to get the next release with the switch to Python3 out asap, there won't be a bugfix release for 0.4.9 (unless something _really, really_ bad turns up that needs immediate fixing, which is extremely unlikely at this point).

But even if, a 0.4.9 bugfix relese will be based on the 0.4.9 release branch, and none of all the changes we commit to master now will affect the 0.4.9 branch anyway.

So, if a potential 0.4.9.1 release is your only concern why you'd want to keep Python2, don't worry. Just remove all the Python2 stuff entirely, I expect that to substantially reduce the potential for errors and headaches... :wink:

Reply:
> I've opened PR: [freeorion/freeorion-sdk#65](https://github.com/freeorion/freeorion-sdk/pull/65)

Ah, good. However, my FO time for today is up, but I should be able to get back to test that PR on Tuesday.

Reply:
@Vezzra I've just got idea we won't be able to merge this PR without python2 in SDK, or I should include python3 migration here as well. Won't it become too big PR with updating to SDK and python3 migration together?

Reply:
As I said again there should be next SDK soon where python2 could be dropped.

Reply:
I also want to mention it was possible to test python3 PR on Tuesday.

Reply:
@o01eg, I have no problem splitting the steps. I agree, doing less things per step and more separate steps is reasonable. If that requires to make a second SDK release soon, I don't see a problem with that.

So I think it is a good idea to first merge this PR with Python2 support still in place, and do the switch to Python3 in a subsequent step.